deploy.php

```diff
@@ -173,10 +193,9 @@ function () {
             return;
         }
 
-        $nvmPath = trim(shell_exec('echo $HOME/.nvm/nvm.sh'));
-
-        if (file_exists($nvmPath)) {
-            runLocally('. ' . $nvmPath . ' && nvm use && nvm exec node_modules/.bin/gulp build');
+        if (commandExist('nvm')) {
```

Comment:
Werkt dit? Ik dacht dat we dat op die andere manier deden omdat nvm een speciaal geval was (bash script) en niet werkte met commandExists.

Reply:
Hm, dan was dat in Uniconstruct nog niet aangepast

Reply:
@dreadnip aangepast in e06f5a300cab56fc4ebfe5251b1ac815aacce38b